applog.go

```diff
@@ -0,0 +1,43 @@
+package mozlogrus
+
+import (
+	"bytes"
+	"encoding/json"
+	"os"
+	"time"
+)
+
+// AppLog implements Mozilla logging standard
+type AppLog struct {
+	Timestamp  int64
+	Time       string
+	Type       string
+	Logger     string
+	Hostname   string `json:",omitempty"`
+	EnvVersion string
+	Pid        int `json:",omitempty"`
+	Severity   int `json:",omitempty"`
+	Fields     map[string]interface{}
+}
+
+// NewAppLog returns a loggable struct
+func NewAppLog(loggerName string, msg []byte) *AppLog {
```

Comment:
add `type string` to the params. It won't break any previous code.

Reply:
added